x-pack/plugins/security_solution/cypress/e2e/exceptions/entry/comments.cy.ts

```diff
@@ -0,0 +1,259 @@
+  esArchiverResetKibana,
+} from '../../../tasks/es_archiver';
+import { login, visitWithoutDateRange } from '../../../tasks/login';
+import {
+  addExceptionFlyoutFromViewerHeader,
+  goToEndpointExceptionsTab,
+  goToExceptionsTab,
+  openEditException,
+  openExceptionFlyoutFromEmptyViewerPrompt,
+  removeException,
+} from '../../../tasks/rule_details';
+import {
+  addExceptionComment,
+  addExceptionConditions,
+  addExceptionFlyoutItemName,
+  clickCopyCommentToClipboard,
+  submitEditedExceptionItem,
+  submitNewExceptionItem,
+  clickOnShowComments,
+  selectOs,
+} from '../../../tasks/exceptions';
+import { DETECTIONS_RULE_MANAGEMENT_URL } from '../../../urls/navigation';
+import { deleteAlertsAndRules } from '../../../tasks/common';
+import {
+  NO_EXCEPTIONS_EXIST_PROMPT,
+  EXCEPTION_ITEM_VIEWER_CONTAINER,
+  EXCEPTION_ITEM_COMMENTS_CONTAINER_TEXT,
+  LOADING_SPINNER,
+} from '../../../screens/exceptions';
+import {
+  createEndpointExceptionList,
+  createExceptionList,
+  createExceptionListItem,
+} from '../../../tasks/api_calls/exceptions';
+import { ROLES } from '../../../../common/test';
+
+describe('Add, copy comments in different exceptions type and validate sharing them between users', () => {
+  describe('Rule exceptions', () => {
+    before(() => {
+      esArchiverResetKibana();
+      esArchiverLoad('exceptions');
+      login();
+      deleteAlertsAndRules();
+      const exceptionList = getExceptionList();
+      // create rule with exceptions
+      createExceptionList(exceptionList, exceptionList.list_id).then((response) => {
+        createRule({
+          ...getNewRule(),
+          query: 'agent.name:*',
+          index: ['exceptions*'],
+          exceptions_list: [
+            {
+              id: response.body.id,
+              list_id: exceptionList.list_id,
+              type: exceptionList.type,
+              namespace_type: exceptionList.namespace_type,
+            },
+          ],
+          rule_id: '2',
+        });
+        createExceptionListItem(exceptionList.list_id, {
+          list_id: exceptionList.list_id,
+          item_id: 'simple_list_item',
+          tags: [],
+          type: 'simple',
+          description: 'Test exception item 2',
+          name: 'Sample Exception List Item 2',
+          namespace_type: 'single',
+          entries: [
+            {
+              field: 'unique_value.test',
+              operator: 'included',
+              type: 'match_any',
+              value: ['foo'],
+            },
+          ],
+        });
+      });
+    });
+    beforeEach(() => {
+      visitWithoutDateRange(DETECTIONS_RULE_MANAGEMENT_URL);
+      goToRuleDetails();
+      goToExceptionsTab();
+    });
+
+    after(() => {
+      esArchiverUnload('exceptions');
+    });
+
+    it('Add comment on a new exception, add another comment has unicode from a different user and copy to clipboard', () => {
+      cy.get(NO_EXCEPTIONS_EXIST_PROMPT).should('not.exist');
```

Comment:
Why this assertion is needed at the beginning of the test?

Reply:
Removed it since we are using the esArchiverResetKibana it will make sure that we start on an empty env